### DISCUSSION TITLE
Drone engage, working but with limited information only

### DISCUSSION THREAD

**Discussion by basmango** [DroneEngage/droneengage_communication#5] [Q&A] [answered]
I am able to observe the mode switches on the web gcs whenever i change the mode in QGroundControl, this works very well, but there are two issues:
1, HUD information is not working, roll pitch yaw are always zero:
2, a lot of logs visible on QGroundControl are not visible on the web GCS

i am connecting droneengage services directly over serial to a cube black on my desk over a UART serial connection, mavproxy works without complaining do let me know how to proceed:
log discrepancy:
![image](https://github.com/user-attachments/assets/39cab720-7b0b-41a4-adca-3cddcebe81fd)
HUD info discrepancy:
![Screenshot from 2024-07-15 13-36-03](https://github.com/user-attachments/assets/dd2d5b54-84ed-42c5-9b7e-ada0fb579123)

**Comment by HefnySco** [comment]
Please make sure that you are running latest versions of the apps [https://cloud.ardupilot.org/downloads/RPI/Latest/](https://cloud.ardupilot.org/downloads/RPI/Latest/)

I tested it with SITL and the online website and it was working fine.
We can make debug-session online if problem is still existing.

**Reply by basmango** [reply]
I had installed the latest version before as well, but i reinstalled it again, still the same issue, no HUD values are being reflected, logs are minimal, but the mode switch and arm status are being reflected instantly.

**Reply by basmango** [reply]
can you confitm if the heading and HUD info are visible even when the GPS is not available? should i be testing with a connected gps?

**Reply by basmango** [reply, marked as answer]
hello i fixed it, swtiched to a larger IO board and tested the connection over USB rather than UART, it was just an issue with my UART connection.
Apologies and thank you for your time!

**Reply by HefnySco** [reply]
Thank you for your feedback.
I still do not understand why the USB is working but not the serial port.... they are all the same from the view point of the app.
if you have more finding please share.
regards
